parm/land/letkfoi/letkfoi.yaml

```diff
@@ -0,0 +1,40 @@
+geometry: $(GEOM_BKG)
```

Comment:
I think we need a specific "field metadata override" within the geometry block for the land DA.

Reply:
@ClaraDraper-NOAA This `field metadata override` is included in the geom dictionary `GEOM_BKG`. We can add the entry `gfs-land.yaml` for our land DA in the yamltool later.

Reply:
`field metadata override: gfs-land.yaml` is added to the yamltools.py for the land DA.